src/test/fixtures/oscap.ts

```diff
@@ -3,14 +3,14 @@ import {
   GetOscapCustomizationsApiResponse,
 } from '../../store/imageBuilderApi';
 
-export const distributionOscapProfiles = (
-  _distribution: string
-): GetOscapProfilesApiResponse => {
+export const distributionOscapProfiles = (): GetOscapProfilesApiResponse => {
   return ['xccdf_org.ssgproject.content_profile_cis_workstation_l1'];
 };
 
 export const oscapCustomizations = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   _distribution: string,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   _profile: string
 ): GetOscapCustomizationsApiResponse => {
```

Comment:
One last small comment, I promise! :sweat_smile: 

We can also get rid of the comments and the unused variables here

```suggestion
export const oscapCustomizations = (): GetOscapCustomizationsApiResponse => {
```